certified-connectors/personr/apiDefinition.swagger.json

```diff
@@ -0,0 +1,492 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "1.0.0",
+        "title": "Personr",
+        "description": "Create, verify and retrieve applicant data using Personr. Effortlessly comply with KYC/AML, using PowerAutomate to integrate into your own technology.",
+        "contact": {
```

Comment:
Hello @charliewesterman, 

The PowerAutomate word cannot be used, could you please remove this, as this is restricted word.